core/src/com/unciv/ui/newgamescreen/GameOptionsTable.kt

```diff
@@ -498,17 +509,22 @@ private class RandomNationPickerPopup(
         okButton.setPosition(innerTable.width - buttonsOffsetFromEdge, buttonsOffsetFromEdge, Align.bottomRight)
         innerTable.addActor(okButton)
 
+        val switchButton = "OtherIcons/NationSwap".toImageButton(Color.YELLOW)
+        switchButton.onClick { switchAllNations() }
+        switchButton.setPosition(innerTable.width / 2, buttonsOffsetFromEdge + 35, Align.center)
```

Comment:
the `+ 35` is just hardcoded. I couldn't understand why it wasn't aligning it with the other buttons which just use `buttonsOffsetFromEdge`. I tested on horizontal and vertical screens and both look the same.